packages/material-ui-lab/src/ClockPicker/ClockPicker.tsx

```diff
@@ -294,7 +294,6 @@ function ClockPicker<TDate>(props: ClockPickerProps<TDate> & WithStyles<typeof s
         ampmInClock={ampmInClock}
         type={view}
         ampm={ampm}
-        // @ts-expect-error TODO figure out this weird error
```

Comment:
I guess the weird error was something like "TDate is not assignable to TDate" which is why `no-shadow` exists.